### ISSUE TITLE
Confirm if breaks in scikit-image examples was fixed by upgrading scikit-image to dev version

### ISSUE THREAD

**Issue by ThunderShiviah** [ThunderShiviah/AllenBrainAtlasAPI#14]
I had a few problems running various examples in scikit-image. I've upgraded to the dev version to see if that would fix the problems. I need to confirm that it has for the problems below:

* out of bounds index error on the canny and hough transform 
* no data.astronaut() file on the image pyramid
* array size mismatch error on the image pyramid

**Comment by ThunderShiviah** [comment]
The data.astronaut() file is now present. Also, I didn't put this in the original post but I previously could only import canny as skimage.filter. Now I can import as ski.image feature as was written in the examples.

**Comment by ThunderShiviah** [comment]
The array size mismatch error on the image pyramid was not due to the version needing to be updated. The issue was caused by the fact that I had grabbed the example code in which a 512x512 image was recursively downsampled and added to a larger composite image. When I initialized the composite image and then tried to recursively add my brain, the program couldn't figure out how to 'broadcast' the dimensions of the brain images into my background image. The dimensions would not overlay correctly. To fix this I had the program print out only the downsampled image (no composite image) at which point the broadcast problem went away.
